src/components/engineViews/BabylonEngineComponent.tsx

```diff
@@ -74,13 +78,21 @@ export const BabylonEngineComponent = (props: {behaveGraphRef: any, setBehaveGra
     }, []);
 
     const resetScene = async () => {
-        sceneRef.current?.meshes.forEach(mesh => mesh.dispose())
+        sceneRef.current?.dispose();
+        createScene();
+
         const file = fileInputRef.current!.files![0]
 
         const url = URL.createObjectURL(file);
 
         const container = await SceneLoader.LoadAssetContainerAsync("", url, sceneRef.current, undefined, ".glb");
         container.addAllToScene();
+
+        // Stop all animations by default (only apply based off of play animation node)
+        for (let i = 0; i < (sceneRef.current?.animationGroups?.length ?? 0); i ++) {
```

Comment:
Babylon has a loader plugin to do this (see https://forum.babylonjs.com/t/how-to-prevent-animations-from-auto-starting-by-sceneloader/27945)